dev/devicelab/bin/tasks/flutter_gallery__transition_perf_e2e_ios.dart

```diff
@@ -6,7 +6,7 @@ import 'package:flutter_devicelab/framework/devices.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/tasks/gallery.dart';
 
-Future<void> main() async {
+Future<void> main(List<String> args) async {
   deviceOperatingSystem = DeviceOperatingSystem.ios;
-  await task(createGalleryTransitionE2ETest());
+  await task(createGalleryTransitionE2EBuildTest(args));
```

Comment:
do we ever use these args?

Reply:
We do. See `GalleryTransitionBuildTest` extends `BuildTestTask`, which parses these args. For context, this is how these args look like: https://flutter.googlesource.com/recipes/+/refs/heads/main/recipes/devicelab/devicelab_drone_build_test.py#142

Reply:
I see. so we were never parsing these args before?

Reply:
That's correct. These args are used for build+test tasks only.

Btw: `Linux gallery` tasks have been running in prod for months, example: `Linux_build_test flutter_gallery__transition_perf`